fenix/app/src/androidTest/java/org/mozilla/fenix/ui/SettingsAddonsTest.kt

```diff
@@ -112,6 +115,7 @@ class SettingsAddonsTest {
         }
     }
 
+    // TestRail link: https://testrail.stage.mozaws.net/index.php?/cases/view/561600
     // Installs uBlock add-on and checks that the app doesn't crash while loading pages with trackers
     @SmokeTest
     @Test
```

Comment:
Rename the manual test to match this test?

Reply:
Sure